packages/apps/graph/src/graph/subscription/new-blocks-from-depth.ts

```diff
@@ -1,5 +1,5 @@
 import { prismaClient } from '@db/prisma-client';
-import { createID } from '@utils/global-id';
```

Comment:
Maybe it was left on purpose, but the `createID` function is not being used anywhere

Reply:
Yep I want to keep it since we will need it again if we want to return an ID somewhere